resources/types/virtual_network.go

```diff
@@ -50,9 +52,13 @@ func NewVirtualNetwork(r *VirtualNetwork, resourceId string, vn *resourcespb.Vir
 
 func (r *VirtualNetwork) Validate(ctx resources.MultyContext) (errs []validate.ValidationError) {
 	errs = append(errs, r.ResourceWithId.Validate()...)
-	//if r.Name contains not letters,numbers,_,- { return false }
-	//if r.Name length? { return false }
 	//if r.CidrBlock valid CIDR { return false }
```

Comment:
@goncalo-rodrigues Should I remove this line also? This seams to be implemented.

Reply:
Oh right, missed that one. Yap!